AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,223 @@
+  installBeginReferrer: "utm_source=google-play&utm_medium=referral&utm_campaign=campaign-123",
+  installVersion: "1.0.0",
+  googlePlayInstant: false
+}
+```
+
+```js
+details = {
+  storePlatform: "Apple App Store",
+  attribution: "true",
+  orgId: "40669820",
+  campaignId: "542370539",
+  conversionType: "Download",
+  adGroupId: "542317095",
+  countryOrRegion: "US",
+  keywordId: "87675432",
+  adId: "542317136"
+}
+```
+
+In the event that a web application has referral information that has expired for that specific
+application, or in any other case where acquisition info is missing for the web application,
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  storePlatform: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+One solution for capturing attribution in organic same-domain installations initiated as an user agent (UA) install
+could be the use of query parameters. If the web application after installation
+has listed GET parameters in their URL as attribution information, within the API we can define acquisition related parameters
+to capture such as `c_id` and translate it into the dictionary return.
+
+```js
+// URL: foo.com?c_id=browserInstallationCampaign
+details = {
+  storePlatform: "",
+  campaignId: "browserInstallationCampaign"
+}
+```
+
+This could also be expanded upon to capture a set of acquisition parameters related to browser installation such as the web
+referrer or any parameter of interest that is already being captured in Store attribution.
+
+### Web Install API
+
+Currently UA installs are initiated by the user directly on the website they want to install.
+There's no opportunity for these web app installs to have attribution information in the same way that store
+initiated web app installations do. However, with the addition of the Web Install API, there exists a
+possibility of cross-domain UA installations that may have associated attribution information through an app directory,
+and the new capability of webapps to install other webapps. Apps installed through the Web Install API will hold a promise
+within its returned success value that allows an opportunity for the installation origin to retrieve attribution information,
+at which point can be captured by the Acquisition Info API. _[See the Web Install API explainer for more information.][Web Install API]_
+
+## Privacy and Security Considerations
+
+1. _Lack of validation_: It is assumed that the Store platform does not perform any validation
+  or sanitation of the Campaign ID itself.
```

Comment:
This wouldn't be limited to the Campaign ID right, but rather _any_ of the data that is stored in the properties.